openshift-hack/cmd/k8s-tests-ext/disabled_tests.go

```diff
@@ -0,0 +1,226 @@
+			"[sig-node] crictl should be able to run crictl on the node",
+
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1953478
+			"[sig-storage] Dynamic Provisioning Invalid AWS KMS key should report an error and create no PV",
+
+			// https://issues.redhat.com/browse/OCPBUGS-34577
+			"[sig-storage] Multi-AZ Cluster Volumes should schedule pods in the same zones as statically provisioned PVs",
+
+			// https://issues.redhat.com/browse/OCPBUGS-34594
+			"[sig-node] [Feature:PodLifecycleSleepAction] when create a pod with lifecycle hook using sleep action valid prestop hook using sleep action",
+
+			// https://issues.redhat.com/browse/OCPBUGS-38839
+			"[sig-network] [Feature:Traffic Distribution] when Service has trafficDistribution=PreferClose should route traffic to an endpoint that is close to the client",
+		},
+		// tests that need to be temporarily disabled while the rebase is in progress.
+		"RebaseInProgress": {
+			// https://issues.redhat.com/browse/OCPBUGS-7297
+			"DNS HostNetwork should resolve DNS of partial qualified names for services on hostNetwork pods with dnsPolicy",
+
+			// https://issues.redhat.com/browse/OCPBUGS-45275
+			"[sig-network] Connectivity Pod Lifecycle should be able to connect to other Pod from a terminating Pod",
+
+			// https://issues.redhat.com/browse/OCPBUGS-17194
+			"[sig-node] ImageCredentialProvider [Feature:KubeletCredentialProviders] should be able to create pod with image credentials fetched from external credential provider",
+
+			// https://issues.redhat.com/browse/OCPBUGS-45214
+			// Even though this feature is not GA in k/k, it will be GA in OCP 4.19, so we should fix it and unskip this test
+			"[Feature:volumegroupsnapshot]",
+
+			// https://issues.redhat.com/browse/OCPBUGS-45273
+			"[sig-network] Services should implement NodePort and HealthCheckNodePort correctly when ExternalTrafficPolicy changes",
+
+			// https://issues.redhat.com/browse/OCPBUGS-45274
+			// https://github.com/kubernetes/kubernetes/issues/129056
+			"[sig-node] PodRejectionStatus Kubelet should reject pod when the node didn't have enough resource",
+		},
+		// tests that may work, but we don't support them
+		"Unsupported": {
+			"[Driver: rbd]",             // OpenShift 4.x does not support Ceph RBD (use CSI instead)
+			"[Driver: ceph]",            // OpenShift 4.x does not support CephFS (use CSI instead)
+			"[Driver: gluster]",         // OpenShift 4.x does not support Gluster
+			"Volumes GlusterFS",         // OpenShift 4.x does not support Gluster
+			"GlusterDynamicProvisioner", // OpenShift 4.x does not support Gluster
+
+			// Skip vSphere-specific storage tests. The standard in-tree storage tests for vSphere
+			// (prefixed with "In-tree Volumes [Driver: vsphere]") are enough for testing this plugin.
+			// https://bugzilla.redhat.com/show_bug.cgi?id=2019115
+			"[sig-storage].*[Feature:vsphere]", //TODO: this rule is broken due to the regex, but it wouldn't pick anything up anyways, safe to remove?
```

Comment:
Note the TODO here, what should we do about this one?

Reply:
They existed in 4.15 [1], but seem to be gone in 4.16. Feel free to drop it. See? We need a `MustSelect` :D

[1] https://github.com/bertinatto/kubernetes/blob/ff493be70058d7324491442be6d57c92cfccb92d/test/e2e/storage/vsphere/persistent_volumes-vsphere.go#L37

Reply:
I do think that will be useful, but can we do something like that in a followup?

Reply:
Definitely